SWLOR.Game.Server/Feature/RecipeDefinition/EngineeringRecipeDefinition/ModuleRecipes.cs

```diff
@@ -176,7 +176,61 @@ private void Tier1()
 				.Component("ref_tilarium", 5)
 				.Component("elec_ruined", 3);
 
-		}
+            // Hypermatter Injector 1
+            _builder.Create(RecipeType.FuelInjector1, SkillType.Engineering)
+                .Category(RecipeCategoryType.ShipModule)
+                .Resref("cap_inject1")
+                .Level(9)
+                .Quantity(1)
+                .RequirementPerk(PerkType.StarshipBlueprints, 1)
+                .EnhancementSlots(RecipeEnhancementType.Module, 1)
+                .Component("ref_tilarium", 5)
+                .Component("elec_ruined", 3);
+
+            // Mirrored Plating 1
+            _builder.Create(RecipeType.ThermalArmor1, SkillType.Engineering)
+				.Category(RecipeCategoryType.ShipModule)
```

Comment:
Bunch of weird formatting with the tabs here.

Reply:
I think I've fixed the tabs now, please let me know if not, but none of this weirdness was showing for me in VS in the first place.